pkg/validator/handler.go

```diff
@@ -77,6 +77,8 @@ func writeResponse(validator Validator, writer http.ResponseWriter, response *v1
 	if _, err := writer.Write(resp); err != nil {
 		validator.Log("level", "error", "message", "unable to write response", microerror.JSON(err))
 	}
+
+	validator.Log("level", "info", "message", fmt.Sprintf("Validating request responded with result: %t", response.Allowed))
```

Comment:
```suggestion
	validator.Log("level", "info", "message", fmt.Sprintf("validated request responded with result: %t", response.Allowed))
```
Isn't the validation already done at this point when there's result? Hence past-tence :slightly_smiling_face: 

I was also wondering if this is useful `info` level log or rather a `debug` one?

Also: Above line 78 seems to miss `"stack"` key before error value.

Reply:
good catch, thank you